djangocms_blog/search_indexes.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+from aldryn_search.helpers import get_plugin_index_data
+from aldryn_search.utils import get_index_base, strip_tags
+from haystack import indexes
+
+from .models import Post
+from .settings import get_setting
+
+try:
+    from django.utils.encoding import force_text as force_unicode_or_text
```

Comment:
this is not needed. Just use ``from django.utils.encoding import force_text`` it exists in every supported Django version